### PR TITLE
Combined packaging of operator deployment permutations

### PR DESCRIPTION
**Pull request by ampie**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-operator-bundle).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-operator-bundle/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 4ba4a3504f6804d24fa8512d37ace9a01ba832db | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | a4d577c079c4de2981faeda852f82a38e539bfc3 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | a4d577c079c4de2981faeda852f82a38e539bfc3 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: requirements.yaml
Comment (joewhite101) [inline requirements.yaml:4]: Looks hardcoded. Is this correct?

**Review by joewhite101 — DISMISSED** [review]
One question to confirm

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-k8s-operator-bundle/pull/11#pullrequestreview-634282877" title="Approved">joewhite101</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-operator-bundle).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-operator-bundle/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-operator-bundle).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-operator-bundle/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-operator-bundle).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-operator-bundle/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 6a33db5c57d9060bd7288279b686cd61328faa0c | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: requirements.yaml
Comment (ampie) [reply requirements.yaml:4]: Unfortunately I had to make changes locally to the entando-k8s-controller-coordinator Helm template. They will be merged into master once we tackle ENG-2227 which would parameterize image version. Once this is in place, we control the final release almost completely from entando-k8s-operator-bundle. I will then also fix this line. 

Note that we will probably not be running these helm templates from a pipeline at any stage as it is quite an important step of our release process.

**Pull request merged**
